intro.tex

```diff
@@ -0,0 +1,15 @@
+
+\section{Introduction}
+\label{sec:intro}
+
+The Verification Control Document (VCD) is a detailed look at where we are in verifying our requirements e.g. \citeds{LSE-61}.
+We verify requirements by
+running tests, the results of which are recorded in Jira.  A description of how the verification testing is set up is given in Section 2 of \citeds{LDM-503}.
+Specifically Section 2.3.2 of \citeds{LDM-503} describes the verification elements, test cases which appear in \secref{sec:vcd}.
+
+In general the summary provided in \secref{sec:summary} will give the reader the overview required, the remainder of the document is a large matrix of all the DM requirements , their verification elements, tests and if those tests have run.
+
+The matrix is fully linked --- any give Test Case mentioned in the matrix such as \jira{LVV-T362} provide a link to the Jira element defining that test case. Similarly relevant documentation such as test reports like \citeds{DMTR-111} are linked to the bibliography which provides a link to the document.
+If you are browsing with Acrobat it is extremely useful to know that if you follow a hyperlink in the document the  apple $\leftarrow$\footnote{This should be ctrl $\leftarrow$ on windows/Linux} takes you back to where you clicked.
+
+Hence this is not a document you want to read but provides a good view on the Jira system in which all of the same information may be found (apart from the summaries).
```

Comment:
"Hence this is not a document you want to read but" → "This reference document is not designed for casual perusal but instead".